streaming/base/storage/download.py

```diff
@@ -272,6 +273,29 @@ def download_from_oci(remote: str, local: str) -> None:
     os.rename(local_tmp, local)
 
 
+def download_from_hf(remote: str, local: str) -> None:
+    """Download a file from remote Hugging Face to local.
+
+    Args:
+        remote (str): Remote path (Hugging Face).
+        local (str): Local path (local filesystem).
+    """
+    from huggingface_hub import hf_hub_download
+
+    if 'hf://' not in remote:
```

Comment:
Can you please use the below code instead as it is inline with other cloud provider codebases?

```
obj = urllib.parse.urlparse(remote)
    if obj.scheme != 'hf':
```